app/controllers/users/movies/viewing_parties_controller.rb

```diff
@@ -1,4 +1,40 @@
 class Users::Movies::ViewingPartiesController < ApplicationController
+  before_action :get_user
   def new
+    @facade = MovieFacade.new(params[:movie_id])
+    @users = User.other_users(@user.id)
```

Comment:
Nice - this was a great catch, glad we're handling excluding the host from the list of partygoers.